CHANGELOG.md

```diff
@@ -22,8 +22,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
- - **CUMULUS-2124**
-   - cumulus-rds-tf terraform module now takes engine_version as an input variable.
+- **CUMULUS-2296**
+  - Fixed parsing of PDR DATA_VERSION by updating the `@cumulus/pvl` package to parse fields more correctly.
```

Comment:
Can we be more specific here?

Reply:
I suppose the specificity is in the ticket. 

Might be worth at least a specific example for users without access to Jira

Reply:
4aa8a 👍